src/Components/Button.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components'
+
+export default function Button({buttonName = '', onClickFunction, message = '', circularButton = false, colorIndex = 0}){
+    
+    const colorsActive = ['#275ef9', '#6120f9', '#e7394d']
```

Comment:
Bei deinen Farben könntest du langfristig mit Variablen in der Global Sytels arbeiten, dann bist du da flexibler, falls du sie mal ändern möchtest.